Library/Services/TopicService.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/*
+ * This file is part of the Disturb package.
+ *
+ * (c) Matthieu Ventura <mventura@voyageprive.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Vpg\Disturb\Services;
+
+class TopicService
```

Comment:
shall we suffix service classes by `Service`

Reply:
#30